src/server/gc_worker/rawkv_compaction_filter.rs

```diff
@@ -135,8 +136,6 @@ impl Drop for RawCompactionFilter {
     fn drop(&mut self) {
         self.raw_gc_mvcc_deletions();
 
-        self.engine.sync_wal().unwrap();
```

Comment:
sync_wal is not needed in v2. But what about v1? I suppose src/server/gc_worker/rawkv_compaction_filter.rs is for v1 as well.

Reply:
rawkv doesn't write to kv db directly, it was added by accident.